config/x_hh_bbtautau_run2.yaml

```diff
@@ -24,16 +28,1318 @@ processes:
                     1000, 1250, 1500, 1750, 2000, 2500, 3000,  ]
     is_signal: true
     scale: 2 * 6.272E-02 * 5.824E-01
-  - TT
-  - DY
+  - process: TT
+  - process: DY
+  - process: VV
+    allow_zero_integral: True
+    allow_negative_bins_within_error: True
+    subprocesses:
+    - WW
+    - WZ
+    - ZZ
+  - process: ST
+    allow_zero_integral: True
+    allow_negative_bins_within_error: True
+    subprocesses:
+    - ST_tW_antitop_5f_InclusiveDecays
+    - ST_tW_top_5f_InclusiveDecays
+    - ST_t-channel_antitop_4f_InclusiveDecays
+    - ST_t-channel_top_4f_InclusiveDecays
+  #- process: TW
+  - process: TTX
+    allow_zero_integral: True
+    allow_negative_bins_within_error: True
+    max_n_sigma_for_negative_bins: 10
+    subprocesses:
+    - TTWW
+    - TTWZ
+    - TTWH
+    - TTZH
+    - TTZZ
+    - TTGG
+    - TTWH
+    - TTZH
+    - TTTJ
+    - TTTT
+    - TTTW
+    - TT4b
+  - process: W
+  - process: EWK
+    allow_zero_integral: True
+    allow_negative_bins_within_error: True
+    subprocesses:
+    - EWK_WplusToLNu
+    - EWK_WminusToLNu
+    - EWK_ZTo2L
+    - EWK_ZTo2Nu
+  - process: VVV
+    allow_zero_integral: True
+    max_n_sigma_for_negative_bins: 1.01
+    allow_negative_bins_within_error: True
+    subprocesses:
+    - WWW_4F
+    - WWZ_4F
+    - WZZ
+    - ZZZ
+  - process: QCD
+    allow_zero_integral: True
+    allow_negative_integral: True
+    allow_negative_bins_within_error: True
+    max_n_sigma_for_negative_bins: 10
```

Comment:
o.O

Reply:
eh for QCD boosted there are issues of negative bins [(e.g. the content is a negative number of o(~1) and the error is o(~10^(-2)) ].. To make a patch I put this.